lua/octo/pickers/telescope/provider.lua

```diff
@@ -767,76 +767,110 @@ end
 --
 -- ASSIGNEES
 --
+local function get_user_requester()
```

Comment:
This function seem to be duplicated for the pickers and the next function would have to be duplicated as well. Is there a file location that would serve to house this logic?

Reply:
Good question, we could create a `pickers/shared` for this kind of code